.github/workflows/test_cov.yml

```diff
@@ -0,0 +1,29 @@
+name: test_cov
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r requirements.txt
+        pip install -r requirements-dev.txt
+    - name: Test
+      run: make test
+    - name: Upload coverage to Codecov
+      uses: codecov/codecov-action@v2
+      with:
+        directory: ./coverage/reports/
+        env_vars: OS,PYTHON
+        fail_ci_if_error: true
+        files: ./coverage.xml
+        flags: unittests
+        verbose: true
```

Comment:
It seem the same as `uploading coverage report setp` in `pypi` workflow. Why do you write it twice ?

Reply:
Duplicate indeed. Because i don't want to run all pypi steps, and it can't be disabled in the workflow. So i use an extra file to test and forget delete. 

I'll delete this file and add `[![codecov](https://codecov.io/gh/wechaty/python-wechaty/branch/main/graph/badge.svg)](https://codecov.io/gh/wechaty/python-wechaty) ` [![codecov](https://codecov.io/gh/helloocc/python-wechaty/branch/testcase/graph/badge.svg)](https://codecov.io/gh/helloocc/python-wechaty) (the icon is my fork repo result）in `README`.